boss.yml.example

```diff
@@ -40,4 +40,16 @@ notifications:
         #   endpoint: /xxxxxxxxx/xxxxxxxxx/xxxxxxxxxxxxxxxxxxxxxxxx
         #
         endpoint: SLACK_ENDPOINT_HERE
-
+    hipchat:
+        enabled: true
+        # 
+        # Whether this message should trigger a user 
+        # notification (change the tab color, play a sound,
+        # notify mobile phones, etc).
+        notify: true
+        #
+        # Hipchat path that identifies your chat room.
+        # For example:
+        #   endpoint: https://companyname.hipchat.com/v2/room/xxxxxx/notification?auth_token=xxxxxx
+        #
+        endpoint: HIPCHAT_ENDPOINT_HERE
```

Comment:
Instead of having to put the full hipchat url here. I think it'd be more convinient for the user to just provide the `authToken` and the `roomId` values in the config. 

We've done it in another project like this:
https://github.com/leapfrogtechnology/chill/blob/master/chill.yml.dist#L15-L17
```yml
  hipchat:
    enabled: false
    authToken: HIPCHAT_WEBHOOK_TOKENID
    roomId: HIPTCHAT_WEBHOOK_ROOMID
```
And in the code we're using those like this:
```javascript
  const url = baseUrl + `${roomId}/notification?auth_token=${authToken}`;

  logger.info('Sending notification to hipchat.');
  logger.debug('Hipchat Payload:', payload);

  return rp.post({
    url,
    json: true,
    body: payload
  });
```

https://github.com/leapfrogtechnology/chill/blob/master/src/services/hipchat.js#L63-L72